articles/how-kubernetes-pods-work/index.md

```diff
@@ -0,0 +1,65 @@
+In [cloud computing](https://www.section.io/engineering-education/introduction-to-cloud-computing/), a pod is a high-level structure and the tiniest execution object in [Kubernetes](https://www.section.io/engineering-education/what-is-kubernetes/). These pods are temporary such that if one fails, Kubernetes simultaneously models a copy such that the work keeps going on. They are usually in one or several collections.
+
+Today, the usage of Kubernetes has become the go-to solution in cloud software management. This is the case because Kubernetes use pods, which is do not run containers directly. As long as a container shares the same pod with another container, they can access the local network and resources.
+
+This article discusses Kubernetes Pods, their model types, benefits, how they work, and how they communicate with each other in Kubernetes.
+
+### What are Kubernetes Pods?
+
+Before we get to what exactly are Kubernetes pods, it would help if you briefly understand some definitions of the resources inside a Kubernetes system.
+
+First off, there is the node, a unit of computing hardware in a cluster. It is essentially one machine in the whole arrangement called a cluster. The cluster is the linked hardware machines under one network creating one machine. So much so, it becomes one powerful device as a whole such that in case one node fails, the work is seamlessly divided among the remaining fully functional nodes in the cluster, and the system remains optimally operating.
```

Comment:
```suggestion
The node is a unit of computing hardware in a cluster. It is essentially one machine in the cluster. The cluster refers to the linked hardware nodes under one network creating one machine. So much so, it becomes one powerful device as a whole. In case one node fails, the work is seamlessly divided among the remaining fully functional nodes in the cluster, and the system remains optimally operating.
```